tests/nightly/test_kvstore.py

```diff
@@ -55,10 +60,160 @@ def test_kvstore(kv_type):
             err = sum(err) / np.sum(np.abs(res[j]))
             assert(err < 1e-6), (err, shapes[j])
 
+def test_compress_kvstore(kv_type, compression='2bit', threshold=0.5):
```

Comment:
Is there a test in which you set compression to active/inactive and verify the result?

Reply:
added